app/controllers/users_controller.rb

```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+  def index; end
```

Comment:
I might personally just make this a regularly formatted `index` block with nothing in it for readability/convention :)